test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -1020,6 +1018,18 @@ func runWorkload(ctx context.Context, b *testing.B, tc *testCase, w *workload) [
 	return dataItems
 }
 
+func maybeCreateNamespace(ctx context.Context, b *testing.B, client clientset.Interface, namespace string, podsPerNamespace *map[string]int) {
```

Comment:
enough?

```suggestion
func createNamespace(ctx context.Context, b *testing.B, client clientset.Interface, namespace string, podsPerNamespace *map[string]int) {
```

Reply:
It doesn't always create the namespace.

`ensureNamespace`?

Reply:
I like `createNamespaceIfNotPresent`, but up to you.

Reply:
Renamed to `createNamespaceIfNotPresent`.